### ISSUE TITLE
ClearTable doesn't handle tables with spaces in their names properly

### ISSUE THREAD

**Issue by darrencomeau** [VulcanAnalytics/DBTester#51]
Does appear to work in many cases but when there is another table which matches the first part of the table name.

``` C#
            dbTester.ClearTable("Weather", "Observation");
            dbTester.ClearTable("Weather", "ICAO Aerodrome"); //Fine
            dbTester.ClearTable("Weather", "Filtered Aerodrome"); // Fine
            dbTester.ClearTable("Weather", "Filter");
            dbTester.ClearTable("Weather", "Aerodrome");
            dbTester.ClearTable("Weather", "Observation Source"); //Problem because of earlier table
```
![image](https://user-images.githubusercontent.com/28704320/51433149-718a1100-1c3c-11e9-8ad8-9f6ea6e7a74d.png)

## Work Around
Use square brackets to quote the table
``` C#
            dbTester.ClearTable("Weather", "[Observation Source]"); //Problem because of earlier table
```

**Comment by darrencomeau** [comment]
NB work around doesn't work in `1.0.0.Beta`! While the code executes without error it doesn't actually do anything.


Method in `1.0.0.Beta2` uses different approach and correctly deletes data from the table with the work around.

**Comment by darrencomeau** [comment]
New test scenario `Will_Clear_Data_From_Table_With_Foreign_Key_And_Space_In_Object_Name` replicates the error
![image](https://user-images.githubusercontent.com/28704320/51869197-c10cc300-2347-11e9-8116-9bd1d490963e.png)

**Comment by darrencomeau** [comment]
Fixed and ready for release in 1.0.0-beta3
